pkg/api/grpcServer.go

```diff
@@ -79,16 +79,21 @@ func (s *grpcServer) Check(legacyCtx legacyContext.Context, req *mixerpb.CheckRe
 
 	globalWordCount := int(req.GlobalWordCount)
 
-	glog.Info("Dispatching Preprocess")
+	if glog.V(1) {
```

Comment:
The trade-off is that the `if glog.V(foo) { ... }` style avoids evaluating arguments if logging is at a lower level than you're checking, while you'll always eval arguments in the `glog.V(foo).Info(bar)` style.